runtime/integration-tests/src/integrations_mock/test_common.rs

```diff
@@ -0,0 +1,985 @@
+// Copyright 2020-2023 Manta Network.
+// This file is part of Manta.
+//
+// Manta is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Manta is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Manta.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Calamari Parachain Integration Tests.
+
+#![cfg(test)]
+#![allow(clippy::identity_op)] // keep e.g. 1 * DAYS for legibility
+
+use super::{mock::*, *};
+use frame_support::{
+    assert_err, assert_noop, assert_ok,
+    traits::{tokens::ExistenceRequirement, Get, PalletInfo},
+};
+use runtime_common::test_helpers::{
+    self_reserve_xcm_message_receiver_side, self_reserve_xcm_message_sender_side,
+    to_reserve_xcm_message_receiver_side, to_reserve_xcm_message_sender_side,
+    ADVERTISED_DEST_WEIGHT,
+};
+
+use manta_primitives::{
+    assets::{
+        AssetConfig, AssetLocation, AssetRegistryMetadata, AssetStorageMetadata, FungibleLedger,
+        FungibleLedgerError,
+    },
+    constants::time::{DAYS, HOURS},
+    types::{AccountId, Balance, CalamariAssetId},
+};
+use session_key_primitives::util::unchecked_account_id;
+use xcm::{
+    opaque::latest::{
+        Junction::{PalletInstance, Parachain},
+        Junctions::X2,
+        MultiLocation,
+    },
+    VersionedMultiLocation,
+};
+use xcm_executor::traits::WeightBounds;
+
+use super::{ALICE, ALICE_SESSION_KEYS};
+use sp_core::sr25519;
+use sp_runtime::{DispatchError, ModuleError};
+
+// currently, we ignore all parachain staking tests in integration tests
```

Comment:
Are these still being called somewhere else?

Reply:
Do you mean these imports? or the ignored test cases?

Reply:
Ignored tests

Reply:
No, they're not being called, Adam told me that these parachain-staking test cases doesn't make sense here with no finalized related staking params, so I followed his advice to ignore them.

Reply:
Ok , could add feature flags to separate calamari and manta if we use different parameters.

Perhaps a future PR

Reply:
Sure.